doc/middleware-it-registratore-telematico/reference-tables/type-of-receipt-ftreceiptcase.md

```diff
@@ -0,0 +1,29 @@
+---
+slug: /poscreators/middleware-doc/italy/reference-tables/ftreceiptcase
+title: 'Type of receipt: ftReceiptCase'
+---
+
+# Type of Receipt: ftReceiptCase
+
+The `ftReceiptCase` indicates the receipt type and defines how the fiskaltrust.SecurityMechanism should process it following Italian law.
```

Comment:
```suggestion
The `ftReceiptCase` indicates the receipt type and defines how the fiskaltrust.SecurityMechanism should process it following italian law.
```

Reply:
is "fiskaltrust.SecurityMechanism" still a term we use?

Reply:
i think it is. i think what it means is the basic chaining of the receipts we are doing in the queue for every market. but however i am not sure ^^